Sources/SwiftSyntax/Convenience.swift

```diff
@@ -45,3 +45,56 @@ extension MemberAccessExprSyntax {
     )
   }
 }
+
+extension EnumCaseParameterSyntax {
+
+  /// Creates an `EnumCaseParameterSyntax` with a `firstName`, and automatically adds a `colon` to it.
+  ///
+  /// - Parameters:
+  ///   - leadingTrivia: Trivia to be prepended to the leading trivia of the node’s first token. If the node is empty, there is no token to attach the trivia to and the parameter is ignored.
+  ///   - colon: If the parameter has a label, the colon separating the label from the type. SInce `firstName` is not nil, the default `colon` will be set to `TokenSyntax.colonToken()`.
+  ///   - type: The parameter's type.
+  ///   - defaultValue: If the parameter has a default value, the initializer clause describing the default value.
+  ///   - trailingComma: If the parameter is followed by another parameter, the comma separating them.
+  ///   - trailingTrivia: Trivia to be appended to the trailing trivia of the node’s last token. If the node is empty, there is no token to attach the trivia to and the parameter is ignored.
+  public init(
+    leadingTrivia: Trivia? = nil,
+    _ unexpectedBeforeModifiers: UnexpectedNodesSyntax? = nil,
```

Comment:
I believe this handwritten initializer is specifically designed for manual use by developers. Given that, I'm wondering if there is the need to include unexpected nodes in it. It doesn't seem likely that anyone would want to manually create an unexpected node. What are your thoughts?

Reply:
That's a _great_ question. I'm actually clueless yet, and just learning. As I work with swift-syntax some more, I hope to understand it better, and actually have an opinion. But alas, not yet.

So I appreciate your feedback and direction on this. Your call. Should I clean them out?